src/System.Text.Json/src/System/Text/Json/Serialization/JsonNamingPolicy.cs

```diff
@@ -9,6 +9,9 @@ namespace System.Text.Json.Serialization
     /// </summary>
     public abstract class JsonNamingPolicy
     {
+        /// <summary>
+        /// Initializes a new instance of <see cref="JsonNamingPolicy"/>.
```

Comment:
I suggest you use the full DocId, like this:

```suggestion
        /// Initializes a new instance of <see cref="T:System.Text.Json.Serialization.JsonNamingPolicy"/>.
```

Reply:
Why? Unless it's necessary is prefer to keep the comment shorter and let the compiler resolve it.

Reply:
If these references are automatically replaced by the compiler, then sure, they can stay like this.

But if they are not modified, then we should use the full DocId otherwise the dotnet-api-docs build will fail (see cref links should resolve to a valid API). The alternative is to wait for the /// to be ported to dotnet-api-docs and the Docs team can manually request getting these references fixed, but then we have to wait for another Docs build iteration to finish.

Reply:
> If these references are automatically replaced by the compiler

They are.

Reply:
Okay, I'll resolve my suggestions for full DocIds. Thanks.